python/pyspark/sql/tests.py

```diff
@@ -3086,18 +3086,35 @@ class ArrowTests(ReusedPySparkTestCase):
 
     @classmethod
     def setUpClass(cls):
+        from datetime import datetime
         ReusedPySparkTestCase.setUpClass()
+
+        # Synchronize default timezone between Python and Java
+        tz = "America/Los_Angeles"
+        os.environ["TZ"] = tz
```

Comment:
Maybe I am too much worried but shouldn't we keep the original `os.environ["TZ"]` and then restore it back in L3115?

Reply:
Hmm, I was assuming that `os.environ["TZ"]` wouldn't be set, but I guess it could be. I'll fix that.